examples/json-web-token.roc

```diff
@@ -0,0 +1,118 @@
+app [Model, server] { pf: platform "../platform/main.roc" }
+
+import pf.Stdout
+import pf.Http exposing [Request, Response]
+import pf.Jwt
+
+Model : {}
+
+server = { init, respond }
+
+respond : Request, Model -> Task Response [ServerErr Str]_
+respond = \_, _ ->
+
+    # We hardcode the JWT here and ignore the request for simplicity, but normally this is how
+    # you would decode the request body (assuming the token is in the body and not a URL param)
+    # ```
+    # exampleBody : Result (Dict Str Str) _
+    # exampleBody = Http.parseFormUrlEncoded req.body
+    # ```
+    exampleBody : Result (Dict Str Str) _
+    exampleBody =
+        Dict.single "id_token" "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.joReqPNNkWQ8zQCW3UQnhc_5NMrSZEOQYpk6sDS6Y-o"
+        |> Ok
+
+    token : Str
+    token =
+        exampleBody
+        |> Result.try \decodedBody -> decodedBody |> Dict.get "id_token"
+        |> Result.withDefault "TOKEN MISSING"
```

Comment:
Can we get rid of the withDefault here an do a proper error as well?

Reply:
Yeah, I'm thinking I change up the example a bit. It's currently in between a test and a user facing example -- I think we agree it should show an end user the right way to do things.